includes/admin/class-wc-admin-dashboard.php

```diff
@@ -37,6 +37,11 @@ public function init() {
 			wp_add_dashboard_widget( 'woocommerce_dashboard_recent_reviews', __( 'WooCommerce recent reviews', 'woocommerce' ), array( $this, 'recent_reviews' ) );
 		}
 		wp_add_dashboard_widget( 'woocommerce_dashboard_status', __( 'WooCommerce status', 'woocommerce' ), array( $this, 'status_widget' ) );
+
+		// Network Order Widget.
+		if ( is_multisite() ) {
+			wp_add_dashboard_widget( 'woocommerce_network_orders', __( 'WooCommerce network orders', 'woocommerce' ), array( $this, 'network_orders' ) );
```

Comment:
In a case where WC is network activated, would this also work on the network dashboard without other changes? If so, it might be a quick nice addition. Just need to use the hook `wp_network_dashboard_setup` too. No biggy if that won't work.